adi_update_boot.sh

```diff
@@ -476,9 +476,9 @@ restoring_extra_files()
         exit 1
       fi
       echo "u-boot.img restored."
-      echo -e "\nRestoring extlinux/extlinux.conf (specific to cyclone5 projects starting with 2021_r1 release)..."
```

Comment:
Actually this is specific to cyclone5, but also to A10SOC starting with Quartus 20.1 Pro version.